src/main/java/Duke.java

```diff
@@ -1,10 +1,163 @@
+                    System.out.println(todoList.get(markedValue).getStatusIcon() + todoList.get(markedValue).getDescription());
+                    System.out.println(LINES);
+
+                case TODO:
+                    try {
+                        Task newTask = new Todo(userInputNewValue);
+
+                        todoList.add(newTask);
+                        System.out.println(LINES);
+                        System.out.println("Got it. I've added this task: ");
+                        System.out.println(todoList.get(todoList.size() - 1).toString());
+
+                        taskString = todoList.size() == 1 ? " task " : " tasks ";
+
+                        System.out.println("Now you have " + todoList.size() + taskString + "in the list.");
+                        System.out.println(LINES);
+                    }
+                    catch (DukeExceptions e){
+                        System.out.println(LINES);
+                        System.out.println("TODO needs description");
+                        System.out.println(LINES);
+                    }
+
+                    break;
+                case DEADLINE:
+
+                    Task newDeadline = new Deadline(userInputNewValue);
+                    todoList.add(newDeadline);
+
+                    System.out.println(LINES);
+                    System.out.println("Got it. I've added this task: ");
+                    System.out.println(todoList.get(todoList.size() - 1).toString());
+
+                    taskString = todoList.size() == 1 ? " task " : " tasks ";
+
+                    System.out.println("Now you have " + todoList.size() + taskString + "in the list.");
+                    System.out.println(LINES);
+
+                    break;
+                case EVENT:
+
+                    Task newEvent = new Event(userInputNewValue);
+                    todoList.add(newEvent);
+
+                    System.out.println(LINES);
+                    System.out.println("Got it. I've added this task: ");
+                    System.out.println(todoList.get(todoList.size() - 1).toString());
+
+                    taskString = todoList.size() == 1 ? " task " : " tasks ";
+
+                    System.out.println("Now you have " + todoList.size() + taskString + "in the list.");
+                    System.out.println(LINES);
+                    break;
+                case DELETE:
+                    System.out.println(LINES);
+                    System.out.println("Noted. I've removed this task: ");
+                    System.out.println(todoList.get(markedValue).toString());
+
+                    taskString = todoList.size() == 2 ? " task " : " tasks ";
+
+                    System.out.println("Now you have " + (todoList.size() - 1) + taskString + "in the list.");
+                    System.out.println(LINES);
+                    todoList.remove(markedValue);
+                }
```

Comment:
[Code Quality] This whole section is not really friendly to read. Consider extracting the logic in each case for better SLAP and control flow.